pool/database/migration/migration_202312040852.ml

```diff
@@ -0,0 +1,17 @@
+let store_settings_timespans_in_seconds =
+  Sihl.Database.Migration.create_step
+    ~label:"store settings timespans in seconds"
```

Comment:
add previous unit to be able to recalculate the factor (day --> seconds)

```suggestion
    ~label:"store settings timespans in seconds instead of days"
```